.circleci/config.yml

```diff
@@ -69,7 +69,7 @@ jobs:
           no_output_timeout: 20m
           command: |
             build_dir="libtelemetry/build"
-            test_apk_path="outputs/apk/androidTest/debug/libtelemetry-debug-androidTest.apk"
+            test_apk_path="outputs/apk/androidTest/okhttp4/debug/libtelemetry-okhttp4-debug-androidTest.apk"
```

Comment:
Don't we want to keep the same behavior here? I mean having okhttp4 versioned telemetry without any suffixes? Even example apps.

Reply:
Unfortunately, it's generated by Google's Gradle android build tools according to build flavour names and I don't know how to override build outputs.